src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderRepository.kt

```diff
@@ -24,6 +24,6 @@ interface OffenderRepository : JpaRepository<Offender, Long> {
   fun findOffenderWithLatestBookingByNomsId(@Param("nomsId") nomsId: String): Optional<Offender>
 
   @Lock(LockModeType.PESSIMISTIC_WRITE)
-  @Query("SELECT o from Offender o left join fetch o.bookings where o.nomsId = :nomsId and o.id = o.rootOffenderId")
-  fun findOffenderByNomsIdOrNullForUpdate(@NotNull nomsId: String): Optional<Offender>
+  @Query("SELECT o from Offender o where o.nomsId = :nomsId and o.id = o.rootOffenderId")
+  fun findRootOffenderByNomsIdForUpdate(@NotNull nomsId: String): Optional<Offender>
```

Comment:
this is only called when there are no bookings - so there is no need to fetch the bookings